src/view/app/Query/Export/export.tsx

```diff
@@ -14,7 +14,7 @@ export default function ExportPopup({
   selectedRows
 }) {
   const [exportFormat, setExportFormat] = React.useState("");
-  const [radioSelection, setRadioSelection] = React.useState("");
+  const [radioSelection, setRadioSelection] = React.useState('Table');

```

Comment:
better not to use hardcode values. Try using enum value from "DataToExport[DataToExport.Table]".